src/services/importing/aioseo/aioseo-robots-provider-service.php

```diff
@@ -41,10 +41,15 @@ public function get_global_robot_settings( $setting_name ) {
 	public function get_subtype_robot_setting( $mapping ) {
 		$aioseo_settings = \json_decode( \get_option( $mapping['option_name'], '' ), true );
 
-		if ( ! isset( $aioseo_settings['searchAppearance'][ $mapping['type'] ][ $mapping['subtype'] ]['advanced']['robotsMeta'][ $mapping['robot_type'] ] ) ) {
-			return false;
-		}
-
 		return $aioseo_settings['searchAppearance'][ $mapping['type'] ][ $mapping['subtype'] ]['advanced']['robotsMeta'][ $mapping['robot_type'] ];
 	}
+
+	/**
+	 * Retrieves the option where the global robot settings exist.
+	 *
+	 * @return array The option where the global robot settings exist.
+	 */
+	public function get_global_option() {
```

Comment:
Could be a consideration to also move this to the suggested `Aioseo_Helper` to make other classes depend on simpler things if they only need this option but otherwise don't need anything robots specific.